bin/bump.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env sh
+
```

Comment:
I think it would be nice if we could get some usage documentation in here for this.

Just for example;

```bash
# ./bin/bump.sh [<version>]
#   <version> The version release version to increment.
#     default: patch
#     options: major|minor|patch
#
# Increments the package version by 1 across the following files;
# - /gulp/tasks/styles.js
# - /includes/scripts-and-styles.php
#
# EXAMPLE USAGE:
#   ./bin/bump.sh minor
```